openreview/openreview.py

```diff
@@ -1055,6 +1064,23 @@ def to_json(self):
             body['tauthor'] = self.tauthor
         return body
 
+    def get_names(self):
```

Comment:
I like this!

Reply:
I considered calling it just "names()", which do you like better?